udaykalra/portfolio/src/main/webapp/style.css

```diff
@@ -1,5 +1,6 @@
 .bottomRow {
   padding-top: 30px;
+  padding-bottom: 30px;
```

Comment:
Alphabetical ordering...

Actually, I think there's a shorthand property, which is preferred by style.

Reply:
Done